Description: stop using imp module which was removed in Python 3.12
Origin: upstream, commits
 https://chromium.googlesource.com/chromium/src/+/9e0c89a3b5638ba2
 https://chromium.googlesource.com/chromium/src/+/f5f6e361d037c316
Last-Update: 2024-03-17

--- a/mojo/public/tools/mojom/mojom/fileutil.py
+++ b/mojo/public/tools/mojom/mojom/fileutil.py
@@ -3,7 +3,6 @@
 # found in the LICENSE file.
 
 import errno
-import imp
 import os.path
 import sys
 
--- a/mojo/public/tools/mojom/mojom/parse/lexer.py
+++ b/mojo/public/tools/mojom/mojom/parse/lexer.py
@@ -2,7 +2,6 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import imp
 import os.path
 import sys
 
